src/components/ha-cover-controls.js

```diff
@@ -60,6 +60,28 @@ class HaCoverControls extends PolymerElement {
     return new CoverEntity(hass, stateObj);
   }
 
+  computeOpenIcon(stateObj) {
+    switch (stateObj.attributes.device_class) {
+      case "awning":
+      case "curtain":
+      case "gate":
+        return "hass:arrow-expand-horizontal";
+      default:
+        return "hass:arrow-up";
+    }
+  }
+
+  computeCloseIcon(stateObj) {
+    switch (stateObj.attributes.device_class) {
+      case "awning":
+      case "curtain":
```

Comment:
Not sure `curtain` should change, as it can also be a roller curtain so up/down would be more fitting?

Reply:
Good call, will adjust